dissect/target/plugins/os/unix/bsd/citrix/_os.py

```diff
@@ -96,16 +111,18 @@ def ips(self) -> list[str]:
     @export(record=UnixUserRecord)
     def users(self) -> Iterator[UnixUserRecord]:
```

Comment:
Is there no information available regarding uid?

Reply:
When you add a user to Citrix Netscaler, it doesn't really add a 'user' to the OS (like adding an entry in `/etc/shadow`). Instead, an SSH key is added that can be used to login, and when users then enter `shell` they drop into a shell as `root`. So for 'human' users, there is no uid. We could do a `yield from super().users` at the end of this function to try to get results out of the `/etc/shadow` file though. I opted against it for now as their home folder is then interpreted as `/`, and if you have a `.netscaler_cli` file in the root of the filesystem, you get the same bash history for a bunch of bogus users.

Reply:
I ended up iterating through `super().users()` and setting the `.home` to None only if the `.home` would otherwise be `/`.